tests/phpunit/CRM/Pledge/BAO/PledgeTest.php

```diff
@@ -149,7 +149,7 @@ public function testGetPledgeStartDate() {
 
     // Try with fixed date
     $params = array(
-      'pledge_start_date' =>  serialize(array('contribution_date' => '2016-06-10')),
+      'pledge_start_date' => serialize(array('contribution_date' => '2016-06-10')),
```

Comment:
@eileenmcnaughton should I go ahead and switch to using json_encode rather than serialize?